package.json

```diff
@@ -86,6 +87,6 @@
     ]
   },
   "engines": {
-    "node": "10.x"
+    "node": "12.x"
```

Comment:
I would recommend you to remove the "engines", it's only needed in case you have a strong reason for it. I think in this case is not needed.